providers/base/units/ethernet/jobs.pxu

```diff
@@ -275,8 +279,11 @@ _steps:
 _verification:
   Did the SUT wake up from S4?
 plugin: user-interact-verify
+imports: from com.canonical.plainbox import manifest
 requires:
-  sleep.disk == 'supported'
+ sleep.disk == 'supported'
```

Comment:
If the device is listed it supports wake up on LAN and something wrong in the system to show `sleep to disk` and `sleep to memory` is `not supported`; then these test cases will be skipped without any notice. For this kind of condition, I prefer to use manifest without resource job.

Reply:
I think this is because this is the S4 test (hibernation). S3 (suspend) is a bit under, and it does check for `sleep.mem == 'supported'`. So I would assume at least one of these jobs would be run, regardless of the status of the device.

Reply:
@hanhsuan I think here your concern is another big topic: sometimes Checkbox will not run some tests just because the resource job it depend failed for some unexpected reason.
But this resource job com.canonical.certification::sleep, it's quit simple to check the /sys/power/state to get if it contain the string such as "mem" or "disk" . So it would hard to fail.

Reply:
Per @hanhusan 's concern, and talk with Kent, just remove the resource job's check, let the tester to decide should or not run this test. They can always choose to skip it and won't just have no chance to run it if something wrong. Anyway, this is a manual test.

Reply:
> I think this is because this is the S4 test (hibernation). S3 (suspend) is a bit under, and it does check for `sleep.mem == 'supported'`. So I would assume at least one of these jobs would be run, regardless of the status of the device.

@pieqq Since it seems Ubuntu doesn't support hibernate, so I just not include the wol on S4 in the test plan.